src/error.c

```diff
@@ -64,7 +64,9 @@ void screaming_death(char *msg)
 
   /* Continuously send error message */
   while (1) {
-    sbp_send_message(MSG_PRINT, 0, len, (u8*)err_msg, &fallback_write);
+    /* TODO: what do we do here?
+    sbp_send_message(SBP_MSG_PRINT, 0, len, (u8*)err_msg, &fallback_write);
```

Comment:
Because of the new style `sbp_send_message`, I had to take this out - how should it go back in? It kind of needs the state stuff to be passed in. Ideas?

/cc @fnoble @cbeighley @mookerji